src/data/applicationSelectData.ts

```diff
@@ -0,0 +1,283 @@
+  "Sault College",
+  "Seneca College",
+  "Sheridan College Institute of Technology and Advanced Learning",
+  "St Peter's College",
+  "Western Academy Broadcasting College",
+  "Yukon College",
+  "Other",
+].map((v) => ({ value: v, label: v }));
+
+const majors: SelectChoice[] = [
+  "Accounting",
+  "Animal Biology",
+  "Animal Science",
+  "Anthropology",
+  "Applied Human Nutrition",
+  "Art",
+  "Art History",
+  "Biochemistry",
+  "Biodiversity",
+  "Biology",
+  "Biological and Medical Physics",
+  "Biological and Pharmaceutical Chemistry",
+  "Biological Engineering",
+  "Biological Science",
+  "Biomedical Engineering",
+  "Biomedical Toxicology",
+  "Bio-Medical Science",
+  "Chemical Physics",
+  "Chemistry",
+  "Chemical Engineering",
+  "Child Studies",
+  "Civil Engineering",
+  "Classical Studies",
+  "Computer Engineering",
+  "Computer Science",
+  "Criminal Justice and Public Policy",
+  "Ecology",
+  "Economics",
+  "Engineering Systems and Computing",
+  "English",
+  "Environment and Resource Management",
+  "Environmental Engineering",
+  "Environmental Sciences",
+  "Equine Management",
+  "European Studies",
+  "Family Studies and Human Development",
+  "Food, Agricultural and Resource Economics",
+  "Food and Agricultural Business",
+  "Food Industry Management",
+  "Food Science",
+  "French Studies/Études Françaises",
+  "General (B.A.)",
+  "Geography",
+  "Health Science",
+  "History",
+  "Honours Agriculture",
+  "Hospitality and Tourism Management ",
+  "Human Kinetics",
+  "International Development Studies",
+  "Life Science",
+  "Management",
+  "Management Economics and Finance",
+  "Marine & Freshwater Biology",
+  "Marketing Management",
+  "Mathematics",
+  "Mechanical Engineering",
+  "Mechatronics Engineering",
+  "Medicine",
+  "Materials Engineering",
+  "Microbiology",
+  "Molecular Biology & Genetics",
+  "Music",
+  "Nanoscience",
+  "Neuroscience",
+  "Nutrition",
+  "Philosophy",
+  "Physical Science",
+  "Physics",
+  "Plant Science",
+  "Political Science",
+  "Psychology",
+  "Public Management",
+  "Spanish & Hispanic Studies",
+  "Sociology",
+  "Software Engineering",
+  "Theatre Studies",
+  "Theoretical Physics",
+  "Undeclared",
+  "Other",
+].map((v) => ({ value: v, label: v }));
+
+const studyYears: SelectChoice[] = [
+  "1st Year",
+  "2nd Year",
+  "3rd Year",
+  "4th Year",
+  "5th Year",
+  "5+ Year",
+  "Graduate Student",
+  "PHD Student",
+].map((v) => ({ value: v, label: v }));
+
+const ethnicities: SelectChoice[] = [
+  "Black / African American",
+  "Hispanic",
+  "East Asian",
+  "South Asian",
+  "Middle Eastern",
+  "Native American",
+  "White / Caucasian",
+  "Multiple ethnicity / Other",
+  "Prefer not to say",
+].map((v) => ({ value: v, label: v }));
+
+const degrees: SelectChoice[] = [
+  "Bachelors",
+  "Masters",
+  "PhD",
+  "Associate's",
+  "Certificate",
+  "Diploma",
+].map((v) => ({ value: v, label: v }));
+
+const heardFrom: SelectChoice[] = [
```

Comment:
would `referredBy` be a more appropriate name here

Reply:
also why is there both heardFrom and discoveredFrom (misspelled as discoverdFrom)

Reply:
Could you explain this again

Reply:
its called heardFrom here but there's also https://github.com/deltahacks/portal/pull/149/files#r1396215969

Reply:
ok i changed the naming